plot_likert/plot_likert.py

```diff
@@ -156,17 +162,18 @@ def plot_counts(
     if counts_are_percentages:
         xlabels = [str(label) + "%" if label != "" else "" for label in xlabels]
 
-    ax.set_xticks(xvalues)
-    ax.set_xticklabels(xlabels)
+    axes.set_xticks(xvalues)
+    axes.set_xticklabels(xlabels)
     if counts_are_percentages is True:
-        ax.set_xlabel("Percentage of Responses")
+        axes.set_xlabel("Percentage of Responses")
     else:
-        ax.set_xlabel("Number of Responses")
+        axes.set_xlabel("Number of Responses")
 
-    # Control legend
-    plt.legend(bbox_to_anchor=(1.05, 1))
+    # Reposition the legend if present
+    if axes.get_legend():
```

Comment:
I added a check for the legend because `legend=0` was not working as expected and I think it's possible people would want to remove the legend.